### PR TITLE
Fixed - torchscript models not working latest master

### LINKED ISSUE #516: Torchscript models not working with latest master
The latest commit for base_handler has introduced a bug due to which scripted models are not working.
The existing script model related test case was having model file [which is not required] hence sanity test suite had not flagged this issue.

### PR DESCRIPTION
**Pull request by dhaniram-kshirsagar**
## Description

Due to recent PR #444, issue #516 got introduced.
Following were the reasons it didn't get caught -
1. CI doesn't have sanity for trochscripted model
2. Tested fix majorly with external script for gpu and cpu [for both scripted and eager mode models] however end to end testing was done only with eager mode model. This is already getting addressed via in progress PR #458 
3. The UTs have mnist scripted model so got under impression that it is covered however realized that mnist_scripted.mar in UT/IT is for cusotm handler and not default handler!!! 

Fixes #516 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

The fix has been tested on gpu and cpu machine. using recently uploaded torchscripted models. Please see attached logs for details.

- Logs
[issue_516_test.txt](https://github.com/pytorch/serve/files/4902523/issue_516_test.txt)

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Comment by dhaniram-kshirsagar** [comment]
> @dhaniram-kshirsagar Please plan to add some regression tests for this to catch problems in future

@chauhang New test cases for torchscript models have been added via PR #458 hence didn't add any test cases.

**Comment by ashokei** [comment]
@dhaniram-kshirsagar seems file path/existence check is removed; 
you could use "get" on dict with default "" so we can still check file existence if user provides a file https://github.com/pytorch/serve/pull/540

above PR also makes "map_location" single point of setting device type/location. This helps future changes if we add another device.

File: ts/torch_handler/base_handler.py [hunk 1]
```diff
@@ -41,13 +41,9 @@ def initialize(self, ctx):
...
 
-        # model def file
-        model_file = self.manifest['model']['modelFile']
-        model_def_path = os.path.join(model_dir, model_file)
-
         map_location = 'cuda' if torch.cuda.is_available() else 'cpu'
-
-        if os.path.isfile(model_def_path):
+        if 'modelFile' in self.manifest['model']:
+            model_file = self.manifest['model']['modelFile']
```
Comment (maaquib) [inline ts/torch_handler/base_handler.py:46]: Use get or default as suggested by @ashokei

**Comment by dhanainme** [comment]
Merged as #549

**Pull request closed without merging**